mintpy/dem_error.py

```diff
@@ -76,6 +76,8 @@ def create_parser():
     parser.add_argument('-s', '--step-date', dest='stepFuncDate', nargs='*', default=[],
                         help='Date of step jump for temporal deformation model,'+
                              ' i.e. date of earthquake/volcanic eruption')
+    parser.add_argument('--periodic', dest='period', type=float, nargs='+', default=[],
+                        help='periodic functinos of temporal deformation model, default = None')
```

Comment:
Adding this sounds fine to me. Could you use `dest='periodic'` to be consistent with `timeseries2velocity.py`?

Reply:
Regarding the help msg, here the default is `[]`, not None. I would recommend using `(default: %(default)s)` style to avoid this kind of typo.

Reply:
No problem! Thanks! I will update these suggestions.